### PR TITLE
Edited bug for PHP 8.0

### PR DESCRIPTION
**Pull request by Viktor-Truhanovich**
Attempted to call an undefined method named "isBuiltin" of class "ReflectionUnionType".

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2.5",
+        "php": ">=8.0",
```
Comment (fabpot) [inline composer.json:14]: I don't think we should drop PHP 7 compact. Any way to make the code work for all currently supported versions?

Comment (Viktor-Truhanovich) [reply composer.json]: I fixed it

**Review by Viktor-Truhanovich — COMMENTED** [review]
Edited support of PHP7

**Comment by Viktor-Truhanovich** [comment]
> Edited support of PHP7



> Edited support of PHP7

**Pull request closed without merging**